conncheck.go

```diff
@@ -6,7 +6,8 @@
 // License, v. 2.0. If a copy of the MPL was not distributed with this file,
 // You can obtain one at http://mozilla.org/MPL/2.0/.
 
-// +build !windows,!appengine
+// +build linux darwin
```

Comment:
connCheck may work unix-like OSs (e.g. aix, dragonfly, freebsd, netbsd, openbsd, solaris), but I'm not sure.
can anyone check it?

Reply:
At least it cross-compiles. I would add them there as well